config/default/core.entity_view_display.node.hs_basic_page.default.yml

```diff
@@ -52,6 +53,7 @@ third_party_settings:
           context_mapping: {  }
           section_width: decanter-grid
           region_widths: center
+          main_content: main-region
```

Comment:
it would be much easier to make this a true/false value on the layout region instead of checking for strings. If that's possible.

Reply:
There are some reasons for using a single string value instead of multiple booleans:
- Code is simpler with a single value, even if it's a string. Checking / assigning multiple booleans would make the conditionals more complex
- There all three possible values for all sections, regardless of available regions. The values for this field don't necessarily match a region.
- It's easier for admin users to set a single value per section, instead of setting multiple values on each region.